docs/source/federation/migrating-from-stitching.md

```diff
@@ -11,9 +11,17 @@ schema stitching, see [this blog post](https://blog.apollographql.com/apollo-fed
 
 ## Summary of steps
 
-You can (and should) migrate **incrementally** from schema stitching to Apollo Federation.
-To do so, you run an Apollo Server gateway _alongside_ your existing schema-stitching gateway and migrate the services that implement your data graph (**implementing services**)
-one at a time.
+This documentation describes a set of steps for migrating architecture **incrementally** from
+stitching to federation. In order to do so, we rely on the fact that changes necessary for
+federation are completely _backwards compatible_. In other words, services that implement a
+part of the graph (**implementing services**) can be used in both your stitching gateway
+and your Apollo gateway.
+
+A brief explanation of the migration strategy is that we recommend that you begin by modifying
+existing services in place to support the federation specification. You should do this while
+the services are still running to support a stitching gateway. At this point, you can stand up
```

Comment:
Cut the "You should do this" sentence and instead append "while continuing to support schema stitching as well" to the previous sentence